source/features/expand-urls-remove-utms.js

```diff
@@ -0,0 +1,13 @@
+function removeUTMs(url) {
+	return url.replace(/[?&#]utm_.*/, '');
+}
+
+export default function () {
```

Comment:
The feature needs to ensure it's only run once. Now it's run on everything for each new tweet.